### DISCUSSION TITLE
Los Certificados en PDF no muestra las imágenes, alguien que me ayude!

### DISCUSSION THREAD

**Discussion by locurandres** [chamilo/chamilo-lms#5344] [General]
![cert](https://github.com/chamilo/chamilo-lms/assets/74663717/c0d167a3-86b4-4812-a9e0-ae0b0ebb307a)

**Comment by chatonito** [comment]
no estoy seguro, pero revisa que el directorio de las imagenes de los certificados, creo que es en /main/img tenga permisos de lectura  o escritura, posiblemente se deba a que el servidor no los muestra por los permisos. estos archivos son png o jepg, que configuran los datos de los archivos que se muestren.

avisanos si se resuelve, y marcalo como "resuelto" para ayudar a los demás, o proporcionar más apoyo.

espero haber ayudado
saludos
atte
dr. agustin rico guzman
webmaster enseñanza médica CAR

**Reply by locurandres** [reply]
Hola amigo, gracias por contestar, ya le di los permisos pertinentes a la carpeta /img los archivos son .png es el certificado que trae por defecto, pero no me muestra, asi estan el codigo: (src="/main/img/side-bar.png) pero en problema persiste.
